src/mission_start.cpp

```diff
@@ -614,6 +614,987 @@ void mission_start::radio_repeater( mission *miss )
     g->u.i_add( item( "repeater_mod_guide", 0, false ) );
 }
 
-void mission_start::place_book( mission * )
+void mission_start::start_commune(mission *miss)
+{
+ tripoint site = target_om_ter_random("ranch_camp_67", 1, miss, false);
+ tinymap bay;
+ bay.load(site.x * 2, site.y * 2, g->get_levz(), false);
```

Comment:
Note: if this mission started from underground, this line would make the NPCs start undeground too. You probably want to use `site.z` instead of `g->get_levz()`.